src/app/Dashboard/RecentResourceItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+// import { Link } from 'react-router-dom';
+
+
+export default function RecentResourceItem(props) {
+	let TopicItem;
+
+	/**
+	 * This function passes in a topics array and item object from a higher level component
+	 * a forEach loop is called on the topics array to find the name of a resources parent topic
+	 * Once found it is assinged to the TopicItem variable
+	 * @param {{topics:array}} topics 
+	 * @param {{item:object}} item 
+	 */
+
+	const getTopicTitle = (topics, item) => {
+		topics.forEach((topic) => {
+			return topic.id === item.resource.parent ? TopicItem = topic.title : null;
+		
+		});
+	};
+
+	return (
+		<React.Fragment>
+			<ul topics={props.topics} resources={props.resources} className="recent-resources">
+				{props.resources.map((item, index) => {
+					return (
+						<li key={index} className="recent-resource-item">
```

Comment:
Indexes should not be used as keys since they are not stable (i.e. they change often). [See the docs here!](https://reactjs.org/docs/lists-and-keys.html#keys)

Reply:
NOTED! Didn't know that since every mentor I've spoken to uses index as a key. Good to know now. I updated the key to be the resource id.